src/graph.py

```diff
@@ -25,6 +25,7 @@
 from networkx.readwrite import json_graph as nx_json
 import community    # pip install python-louvain
 from sklearn.cluster import SpectralClustering
+from scipy import stats
```

Comment:
If we need scipy, please add that to requirements.txt and setup.py

Reply:
Done